examples/changing_defaults.ipynb

```diff
@@ -0,0 +1,501 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Changing defaults\n",
+    "\n",
+    "Up until now, we've only used the default parameters. Contact Map Explorer always determines contacts based on atom-atom distances, i.e., residues are considered in contact if a pair of an atom from one residue is within a cutoff distance of an atom from another residue. But which atoms? And what is that cutoff distance? These decisions can be customized, and you can get improved performance by customizing them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import mdtraj as md\n",
+    "traj = md.load(\"5550217/kras.xtc\", top=\"5550217/kras.pdb\")\n",
+    "topology = traj.topology"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from contact_map import ContactFrequency"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Customizing the atoms involved in the contact\n",
+    "\n",
+    "`ContactFrequency` takes the parameters `query` and `haystack`, which are lists of atom indices. It will then search for all contacts between atoms in `query` and atoms in `haystack`. This allows you to, for example, focus on the contacts between two distinct parts of a protein. By only including some atoms in the search, the contacts are calculated more quickly.\n",
+    "\n",
+    "This also allows you to fundamentally change the definition of a contact by making it about C$_\\alpha$ or about all atoms, instead of heavy atoms as is the default (though if you change that, you should also change the cutoff value). \n",
+    "\n",
+    "In general, it is easiest to get the list of atom indices from MDTraj using its [atom selection language](https://mdtraj.org/1.9.4/examples/atom-selection.html). The default behavior is to look for contacts between all heavy (i.e., non-hydrogen), non-water atoms."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1408\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the default selection is\n",
+    "default_selection = topology.select(\"not water and symbol != 'H'\")\n",
+    "print(len(default_selection))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the general assumption is that the query is no larger than the haystack. if this isn't obeyed, you'll still get correct answers, but some algorithms may be less efficient, and visualizations have also been designed with this in mind."
+   ]
+  },
```

Comment:
Is this still true? (It has been a while since I thought about this requirement in the code)

If this is still true:
```suggestion
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note that the general assumption is that the query is no larger than the haystack. If this isn't obeyed, you'll still get correct answers, but some algorithms may be less efficient, and visualizations have also been designed with this in mind."
   ]
  },
```
else:
```suggestion
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Note that the general assumption is that the query is no larger than the haystack and visualizations have been designed with this in mind."
   ]
  },
```

Reply:
"***some*** algorithms ***may*** be less efficient" -- I'm very intentionally hedging here. 😆 I'm not saying which algorithms, or even that any actually are less efficient. Really, this is as much to developers: if you have an algorithm that needs one list to be bigger, assume the haystack is bigger. We keep telling the users to act that way, and then devs can take advantage of it.